CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 # Changelog
 
 ## [Unreleased]
+
+## [15.0.4]
```

Comment:
I thought we did the ESLint update as 15.1.0? It seems like we have fallen out of sync here.